automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/notebooks/NotebookGCEClusterMonitoringSpec.scala

```diff
@@ -1,115 +0,0 @@
-package org.broadinstitute.dsde.workbench.leonardo.notebooks
-
-import java.io.File
-
-import org.broadinstitute.dsde.workbench.auth.AuthToken
-import org.broadinstitute.dsde.workbench.leonardo._
-import org.broadinstitute.dsde.workbench.leonardo.rstudio.{RStudio, RStudioTestUtils}
-import org.http4s.AuthScheme
-import org.http4s.headers.Authorization
-import org.openqa.selenium.Keys
-import org.scalatest.{DoNotDiscover, ParallelTestExecution}
-
-import scala.concurrent.duration._
-import scala.util.Try
-
-/**
- * This spec verifies cluster status transitions like pause/resume and cluster PATCH.
- * It is similar in intent to ClusterStatusTransitionsSpec but uses notebooks for validation,
- * so lives in the notebooks sub-package.
- */
-@DoNotDiscover
-class NotebookGCEClusterMonitoringSpec
```

Comment:
This spec only had cases for pause/resume Jupyter and RStudio. That is now covered by `RuntimeGceSpec`, so I removed this file.